### PR TITLE
Upgrade style

### PR DESCRIPTION
**Pull request by ldgeo**
- upgrade style example to latest version
- upgrade mapbox-gl-js
- move index example to its own file
- add documentation to download fonts and serve them with Postile
- remove legacy URL's referencing `tilehosting` webservices
- Centering style to `Lyon` by default

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    <style>
+        body { margin:0; padding:0; }
+        #map { position:absolute; top:0; bottom:0; width:100%; }
+    </style>
+</head>
+<body>
+
+<div id='map'></div>
+<script>
+mapboxgl.accessToken = ' ';
```
Comment (autra) [inline index.html]: do we need an access token here? If yes, maybe mention it in the doc.

**Review by autra — APPROVED** [review]
I didn't test yet, but from what I can see, LGTM :-)

File: index.html (hunk 1 above)
Comment (ldgeo) [reply index.html]: no we don't need access token, i remove it now. thanks.

**Comment by ldgeo** [comment]
Added one more commit about the `-diff` mode used in the `import-osm` container.

**Pull request merged**